src/Configuration/PlatesFormatterConfiguration.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Equip\ContentNegotiation;
+use Equip\Formatter\PlatesFormatter;
+
+class PlatesFormatterConfiguration implements ConfigurationInterface
```

Comment:
Also, since `PlatesFormatter` now uses `withTemplate()` to set its template instead of payload settings, I'm not sure how useful this configuration even is right now. It seems like more configuration is necessary for `PlatesFormatter` to be useful in content negotiation, but that configuration would probably be application-specific.